modules/core/include/opencv2/core/va_intel_interop.hpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_CORE_VA_INTEL_INTEROP_HPP
+#define OPENCV_CORE_VA_INTEL_INTEROP_HPP
+
+#ifndef __cplusplus
+#  error va_intel_interop.hpp header must be compiled as C++
+#endif
+
+#if defined(HAVE_VA_INTEL) && defined(HAVE_OPENCL)
+#  include <CL/cl.h>
+#  ifdef HAVE_VA_INTEL_OLD_HEADER
+#    include <CL/va_ext.h>
+#  else
+#    include <CL/cl_va_api_media_sharing_intel.h>
+#  endif
```

Comment:
It is a public header.
And without properly configured build system all this stuff is not available.

In `va_intel.hpp` we using stubs like `typedef void* VADisplay;`

Reply:
i see

Reply:
Done b56bc32